modules/components/org.wso2.carbon.appfactory.core/src/main/java/org/wso2/carbon/appfactory/core/RemoteRegistryService.java

```diff
@@ -160,4 +162,29 @@ public abstract void copyNonExistingResources(String sourceServerUrl, String sou
 	                                              String destServerUrl, String appId, String username)
 			throws AppFactoryException;
 
+	/**
+	 * Create runtime properties on Kubernetes for a application
```

Comment:
I have created a JIRA for to fix this in allover the code.
https://wso2.org/jira/browse/APPFAC-3516